src/Gir/WebRoot/js/checkout-integration.js

```diff
@@ -471,5 +494,15 @@ const initCheckout = (
       ? deliveryAddressChangedCallback
       : false,
     includePaymentFeeInTotalPrice: includePaymentFeeInTotalPrice,
+    shippingOptionChangedCallback: isShippingOptionChangedCallbackEnabled
+      ? shippingOptionChangedCallback
+      : false,
+    paymentMethodChangedCallback: isPaymentMethodChangedCallbackEnabled
+      ? paymentMethodChangedCallback
+      : false,
+    modeChangedCallback: isModeChangedCallbackEnabled
+      ? modeChangedCallback
+      : false,
```

Comment:
Shouldn't this be undefined, if the callback is not enabled?

Reply:
Should be ok this way but it would work with undefined as well.